logos-codegen/src/mir.rs

```diff
@@ -10,12 +10,28 @@ use crate::error::{Error, Result};
 
 lazy_static! {
     /// DOT regex that matches utf8 only.
-    static ref DOT_UTF8: Hir = Hir::dot(false);
+    static ref DOT_UTF8: Hir = Hir::dot(Dot::AnyChar);
 
     /// DOT regex that matches any byte.
-    static ref DOT_BYTES: Hir = Hir::dot(true);
+    static ref DOT_BYTES: Hir = Hir::dot(Dot::AnyByte);
```

Comment:
I think the comments can be removed here, since `true` and `false` got a meaninful enum so the comment is not necessary anymore

Reply:
Indeed :)